src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -244,6 +230,27 @@ export const hasOtherEvidence = formData =>
 export const hasPrivateEvidence = formData =>
   _.get(DATA_PATHS.hasPrivateEvidence, formData, false);
 
+/**
+ * Inspects all given paths in the formData object for presence of values
+ * @param {object} formData  full formData for the form
+ * @param {array} fieldPaths full paths in formData for other fields that
+ *                           should be checked for input
+ * @returns {boolean} true if at least one path is not empty / false otherwise
+ */
+export const fieldsHaveInput = (formData, fieldPaths) =>
+  fieldPaths.reduce(
```

Comment:
Alternatively:
```suggestion
  fieldPaths.some(path => !!_.get(path, formData, '');
```

Sidebar: GitHub suggestions are cool, but don't work super well for multi-line suggestions, I think. Unless I'm missing something. :man_shrugging: